pkg/operator/starter.go

```diff
@@ -332,6 +334,7 @@ var deploymentConfigMaps = []string{
 	"etcd-serving-ca",
 	"kubelet-serving-ca",
 	"sa-token-signing-certs",
+	"oauth-metadata",
```

Comment:
this will cause kube-apiserver to never rollout until the CR is present because you're requiring it.